library/sha256.c

```diff
@@ -74,8 +74,14 @@ do {                                                    \
 } while( 0 )
 #endif
 
+#define MBEDTLS_SHA256_VALIDATE_RET(cond)                           \
```

Comment:
I think we don't have `MBEDTLS_XXX` names in translation units, do we? What about `SHA256_VALIDATE[_RET]`?

Reply:
I dont think we do, but apparently thats what the PoC wanted: https://github.com/ARMmbed/mbedtls/pull/2268/files#diff-b621d92c71bafecc8ae442a2bff4d1fdR59